src/options/Keybindings/default.json

```diff
@@ -654,6 +654,123 @@
         "clipboardNewWindow": [],
         "clipboardIncognitoWindow": []
       }
+    },
+    {
+      "name": "VimFx",
+      "options": {
+        "physicalKeys": false,
+        "ignoreModifierKeys": false,
+        "toggleHelpMenu": [[{ "code": "Slash", "key": "?", "shiftKey": true }]],
+        "toggleSaka": [],
+        "openLink": [[{ "code": "KeyF", "key": "f" }]],
+        "openLinkInBackgroundTab": [
+          [{ "code": "KeyF", "key": "F", "shiftKey": true }]
+        ],
+        "openLinkInForegroundTab": [{ "code": "KeyE", "key": "e"},{ "code": "KeyT", "key": "t"}],
+        "openLinkInNewWindow": [{ "code": "KeyE", "key": "e"},{ "code": "KeyW", "key": "w"}],
+        "openLinkInIncognitoWindow": [{ "code": "KeyE", "key": "e"},{ "code": "KeyP", "key": "p"}],
+        "downloadLink": [],
+        "focusLink": [],
+        "focusInput": [
+		  [{ "code": "KeyE", "key": "e" }, { "code": "KeyF", "key": "f" }]
```

Comment:
`ef` should be assigned to `focusLink`, not `focusInput`, since `ef` focuses both links and inputs in VimFx (really, anything that is focusable), and `focusLink` does just that.

Reply:
k